scala/Scala.g4

```diff
@@ -157,17 +161,25 @@ infixExpr
    ;
 
 prefixExpr
-   : ('-' | '+' | '~' | '!')? ('new' (classTemplate | templateBody) | blockExpr | simpleExpr1 '_'?)
+   : ('-' | '+' | '~' | '!')? (simpleExpr | simpleExpr1)
+   ;
+
+simpleExpr
+   : 'new' (classTemplate | templateBody)
+   | blockExpr
    ;
 
+// Dublicate lines to prevent left-recursive code.
+// can't use (simpleExpr|simpleExpr1) '.' Id
 simpleExpr1
```

Comment:
Is it possible to use a more clear name?

Reply:
Yes, but in spec https://github.com/scala/scala/blob/2.13.x/spec/13-syntax-summary.md used `SimpleExpr` and `SimpleExpr1`, just want to be compliant with spec.

Reply:
Ok then :)